forms-flow-api/src/formsflow_api/resources/form_process_mapper.py

```diff
@@ -383,3 +383,35 @@ def post():
         except BusinessException as err:
             current_app.logger.warning(err.error)
             return err.error, err.status_code
+
+
+@cors_preflight("PUT,OPTIONS")
+@API.route("/form-update/<string:form_id>", methods=["PUT", "OPTIONS"])
+class FormioFormUpdateResource(Resource):
+    """Resource for formio form Update."""
+
+    @staticmethod
+    @auth.require
```

Comment:
Please secure this endpoint with DESIGNER roles. e.g replace auth.require with auth.has_one_of_roles(['formsflow-designer'])

Reply:
added @auth.has_one_of_roles([DESIGNER_GROUP])